app/lib/view/settings/subsettings/notifications.dart

```diff
@@ -2,208 +2,378 @@ import 'dart:async';
-                      });
-                    }
+                            setState(() {
+                              notificationInterval = val;
+                            });
+                          }
                         : null,
-                    onChangeEnd: allowNotifications
+                    onChangedEnd: notificationsActive
                         ? (val) {
-                      int value = val.round();
-                      accountDatabase.kv.set(
-                          'notifications-android-target-interval-minutes',
-                          value.toString());
-                    }
+                            accountDatabase.kv.set(
+                                'notifications-android-target-interval-minutes',
+                                val.round().toString());
+                          }
                         : null,
+                    label: notificationInterval.round().toString(),
                     min: 15.0,
                     max: 180.0,
+                    divisions: 11,
+                    inactiveColor: sliderColor,
+                  )),
+              SizedBox(
+                height: 8.0,
+              ),
+            ],
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 16.0),
+              child: Text(
+                "Applets",
+                style: Theme.of(context).textTheme.labelLarge!.copyWith(
+                      color: notificationsActive
+                          ? Theme.of(context).colorScheme.primary
+                          : Theme.of(context).colorScheme.onSurfaceVariant,
+                    ),
+              ),
+            ),
+            SizedBox(
+              height: 8.0,
+            ),
+            ...applets.map((key) => Padding(
+                  padding: const EdgeInsets.only(bottom: 4.0),
+                  child: MinimalSwitchTile(
+                    title: Text(
+                      supportedApplets[key]?.label(context) ?? key,
+                    ),
+                    leading: Icon(
+                      supportedApplets[key]?.selectedIcon.icon,
+                    ),
+                    contentPadding: EdgeInsets.symmetric(horizontal: 16.0),
+                    value: (snapshot.data![key] ?? 'true') == 'true',
+                    onChanged: notificationsActive
+                        ? (value) {
+                            sph!.prefs.kv.set(key, value.toString());
+                          }
+                        : null,
+                    useInkWell: true,
                   ),
+                )),
+            SizedBox(
+              height: 12.0,
+            ),
+            if (Platform.isAndroid && widget.accountCount > 1) ...[
+              Divider(),
+              SizedBox(
+                height: 16.0,
+              ),
+              Padding(
+                  padding: const EdgeInsets.only(left: 16.0),
+                  child: SliderTile(
```

Comment:
Could we maybe add some kind of unit (like min) and not just the number.

Reply:
#279